src/styles/blocks/contact-us.scss

```diff
@@ -0,0 +1,79 @@
+.contact-us {
+
+  &__info {
+    display: flex;
+    justify-content: space-around;
+
+    @media (max-width: 520px) {
+      flex-direction: column;
+    }
```

Comment:
Weird. Due to design, it must be a column until the Desktop view.

Reply:
As for me this view for the given resolution just looks better
![image](https://user-images.githubusercontent.com/121725856/218306645-2858e1d8-0034-4c41-89a6-611a2d6db49c.png)
comparing to this
![image](https://user-images.githubusercontent.com/121725856/218306672-c02f6bf3-bc45-47ac-b3c4-6ea482e876cf.png)
but if we have to strictly follow the mockup, thats ok.